nixosConfigurations/services/steward.nix

```diff
@@ -60,7 +58,18 @@ with flake-utils.lib.system; let
       services.steward.package = pkgs.steward.production;
     })
   ] "profian.com" "attest";
+
+  attest-production = mkSteward x86_64-linux [
```

Comment:
This seems to imply that it's a fully resolved OS config, whereas it is a curried function.
How about `mkProductionSteward` right below `mkSteward` above? It could take an integer as an argument and then just do something like this?
```nix
mkProductionSteward = n: mkSteward x86_64-linux [
# modules
] "steward.rdu.infra.profian.com" "prod${n}";
```

Reply:
Sure.